script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -381,30 +382,30 @@ In the background, Noel-sensei has become visibly flustered.
 [sha:ff9eea7b0f7ebcf30cf70dd2c9e48b84a62933ad]{
 -- Page 698, Offset 26146.
 -- アルクェイドの敵がどんな吸血鬼なのか、俺はまだ何も知らないんだから。
-Right now I'm still clueless about what kind of vampire Arcueid's enemy is.
+Right now, I'm still clueless about what kind of vampire Arcueid's enemy is.
 }
 [sha:38edf340f1b2d98672db4132c98193ff44201fa5]{
 -- Page 699, Offset 26147.
 -- 「……俺は助かるけど。いいのか、そんな話をして」
 "...That would really help me out.
-Are you sure it's okay for you to tell me, though?"
+ Are you sure it's okay for you to tell me, though?"
 }
 [sha:1fcf007064202de84a6f27c37986faf39b372bc7]{
 -- Page 699, Offset 26148.
 -- 「よくはねえ。機密事項だ。
-"Of course not. This is confidential.
+"'Course not. This is confidential.
 }
 [sha:5583b1c8ce547430fa2fdf242a6dc78aa0a4bfa5]{
 -- Page 699, Offset 26149.
 -- だがまあ、トオノは当事者だし？　異教徒はともかく、無神論者は厚く遇するのが信条でね。いずれ大事な資金源になる」
-But it concerns you, right?
+But it concerns him, right?
```

Comment:
Hm, why was this change to him?

Reply:
I think because he's addressing Noel but referring to Shiki